### PR TITLE
Refactor some part of helpers and split into related packages

### PR DESCRIPTION
**Pull request by anirudhjain75**
Signed-off-by: Anirudh Jain <anirudh.jain@outlook.com>

**Description**
Making utilities of mesheryctl more manageable and categorized into refined packages

**Note to Reviewers**
Work is required to look into the code for making structs / merged into existing ones for better manageability. For now, Only the code is moved to respective files and ensured that the imported and references are updated accordingly

**[Signed commits](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [.] Yes, I signed my commits.

### CODE REVIEW

**Comment by leecalcote** [comment]
Thank you, @anirudhjain75. Heads-up: there’s a merge conflict.

File: mesheryctl/pkg/utils/healthcheck.go [hunk 1]
```diff
@@ -108,3 +111,49 @@ func parseKubectlShortVersion(version string) ([3]int, error) {
...
+		}
+	case "kubernetes":
+		{
+			client, err := meshkitkube.New([]byte(""))
+
+			if err != nil {
+				return false, errors.Wrap(err, "failed to create new client")
+			}
+
+			//podInterface := client.KubeClient.CoreV1().Pods(MesheryNamespace)
```
Comment (hexxdump) [inline mesheryctl/pkg/utils/healthcheck.go:134]: It will be good to clean up the commented code related to Pod.

**Review by hexxdump — APPROVED** [review]
LGTM

**Comment by metonymic-smokey** [comment]
Thank you for this @anirudhjain75 !
LGTM!

**Review by metonymic-smokey — COMMENTED** [review]
Thank you for this @anirudhjain75 !
LGTM!

**Review by navendu-pottekkat — APPROVED** [review]

**Pull request merged**